### PR TITLE
Add blizzard style

### PR DESCRIPTION
**Pull request by karst**
This PR adds a replacement for Ice style. 

I made this song a while back with my buddy Greyfade. Asked him if it was ok to clean it up and make a loopable version of it and he was a-okay with it. 

Let me know if I need to do any changes.

### CODE REVIEW

**Comment by Gymnasiast** [comment]
I think it might be a bit too bassy. The original were not base-heavy at all, and you’re hearing this song with all kinds of other noises mixed into it. What I fear is that the players will hear mostly bass once in game. (Also keep in mind that most users will have worse audio equipment than you.) I think the melody and treble should probably be more emphasised. And perhaps a few more snowy effects, there are quite a few at the beginning, but not so many after.

I think the basis of the song is still good. It’s okay that it’s more Jalmaan than Allister Brimble - it makes the OpenRCT2 soundtrack interesting in its own right. Just making sure it evokes the same feeling and works in practice.

Hope my rambling makes some sense. ;-)

**Review by tupaschoal — COMMENTED** [review]
Personally I don't love the "vocals" (for lack of knowledge on music) that comes here and there. I think it sounds too fast for something that is snow, I'd expect something somber

**Comment by karst** [comment]
I have heard complaints about this before. Can you check out the 122 bpm version I posted in #open-sound-and-music in discord and see if you like that better?

File: replacements/README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Since the merry-go-round music is a playlist of fairground organ this might requ
 |                           | Rock style 3                | Metal style 2                    | Alternative |
 |                           | Roman fanfare style         | Empire style                     | Alternative |
 |                           | Summer style                | Spring style                     | Alternative |
-|                           | Snow style                  | Winter style                     | Alternative |
+|                           | Snow style                  | Winter style                     | Alternative |
```
Comment (Gymnasiast) [inline replacements/README.md]:
```suggestion
|                           | Snow style                  | Winter style                     | Alternative |
```

**Pull request merged**